pipelines/copy-number-variation-vector/gcp/CNVTasks.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+task ConsolidateHMMOutput {
+    meta {
+        description: "This task takes the output from the HMM subpipeline and combines it into a single tarball."
+    }
+    parameter_meta {
+        hmm_tarballs: "The output files from the HMM sub-pipeline. This is an array of tarballs, one for each sample."
+        output_dir: "The output directory for the tarball."
+    }
+    input {
+        Array[File] hmm_tarballs
+        String output_dir
+        # runtime values
+        String docker = "us.gcr.io/broad-gotc-prod/cnv:1.0.0-1679431881"
+        String ram = "8000 MiB"
+        Int cpu = 16
+        # TODO: Make disk space dynamic based on input size
+        Int disk = 70
+        Int preemptible = 3
+        String runtime_zones
+    }
+    command <<<
+        set -x
+        echo "Current directory: "
+        pwd
+        #For each file in hmm_tarballs, extract the tarball and move the contents to the output directory
+        for tarball in ~{sep=' ' hmm_tarballs} ; do
+        echo "Extracting tarball: " $tarball
+        tar --backup=numbered -zxvf $tarball
+        ls -lht
+        done
+        ls -lht
```

Comment:
might be able to clean up these ls statements, unless you want to leave them for future debugging

Reply:
I left these for debugging